app/controllers/director/camps_controller.rb

```diff
@@ -10,10 +14,13 @@ def new
 
   def create
     @camp = Camp.new(camp_params)
```

Comment:
For the address, would you be able to take advantage of the addresses table? It will require some DB changes